solara/components/input.py

```diff
@@ -197,15 +196,28 @@ def Page():
     * `clearable`: Whether the input can be cleared.
 
     """
+
+    def str_to_float(value: Optional[str]):
+        if value is not None and value != "":
```

Comment:
I think this ideas easier
```suggestion
        if value:
```